uiowa/uiowa
# [Bug] Fix replicate event subscriber for multiple list blocks

Resolves #8279 





# How to test



```
ddev blt ds --site=default && ddev drush @default.local uli /node/1/replicate
```
See that the page is successfully replicated

```
ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /node/51/replicate
```
See that the page is successfully replicated
Try to replicate https://sandbox.uiowa.ddev.site/node/41/replicate

Alternatively

- find or create a page that has at least 2 list blocks within a section, either in the same region (column) or across multiple regions
- `git checkout main`
- Attempt to replicate the page, and receive a WSOD
- `git checkout replicate-hotfix`
- Attempt to replicate the page, and SUCCESS

Further testing for good measure? Try or find different combos of 1, 2, or more list blocks with or with/out other blocks in the section and see that it still doesn't error when trying to replicate, and that blocks are replicated in the proper order
